assets/css/_swings_view.scss

```diff
@@ -56,7 +56,8 @@
   background-color: $color-bg-base;
   position: sticky;
   position: -webkit-sticky;
-  top: -1px;
+  // top offset to stick below the view-header
+  top: 2.5rem;
```

Comment:
One thing: for this, I'd prefer if we had a `$view-header-height` constant or something like that, which can then be referenced both here in `_view_header.scss` or anywhere else that needs to use that value for layout purposes.